### PR TITLE
1.20.1 not use net.minecraftforge.bootstrap.ForgeBootstrap

### PR DESCRIPTION
**Pull request by TonimatasDEV**
Okey, now is ready

### CODE REVIEW

File: src/main/java/org/kettingpowered/launcher/KettingLauncher.java [hunk 1]
```diff
@@ -371,16 +371,7 @@ void launch() throws Exception {
...
-            try {
-                Class.forName(className);
-                return true;
-            } catch (ClassNotFoundException e) {
-                return false;
-            }
-        }).toList().get(0);
-
-        Class.forName(clazz, true, KettingLauncher.class.getClassLoader())
+        Class.forName(Bundled_McVersion.equals("1.20.1") ? "cpw.mods.bootstraplauncher.BootstrapLauncher" : "net.minecraftforge.bootstrap.ForgeBootstrap", true, KettingLauncher.class.getClassLoader())
```
Comment (C0D3-M4513R) [inline src/main/java/org/kettingpowered/launcher/KettingLauncher.java:374]: Bundled_McVersion is only set when bundled. Will cause a NPE

File: src/main/java/org/kettingpowered/launcher/KettingLauncher.java [hunk 2]
```diff
@@ -371,7 +371,7 @@ void launch() throws Exception {
         arg_list.add("--launchTarget");
         arg_list.add(args.launchTarget());
 
-        Class.forName("net.minecraftforge.bootstrap.ForgeBootstrap", true, KettingLauncher.class.getClassLoader())
+        Class.forName(Bundled_McVersion.equals("1.20.1") ? "cpw.mods.bootstraplauncher.BootstrapLauncher" : "net.minecraftforge.bootstrap.ForgeBootstrap", true, KettingLauncher.class.getClassLoader())
```
Comment (C0D3-M4513R) [inline src/main/java/org/kettingpowered/launcher/KettingLauncher.java:374]: Bundled_McVersion is only set when bundled. Will break in production.

**Review by C0D3-M4513R — CHANGES_REQUESTED** [review]

**Comment by C0D3-M4513R** [comment]
Also why change the way minecraft launches? it works just fine rn?

**Comment by TonimatasDEV** [comment]
> Also why change the way minecraft launches? it works just fine rn?

It works but, mods need 1.20.1 libraries

**Comment by C0D3-M4513R** [comment]
no mod should access 1.20.1 bootstrapper honestly.

**Comment by C0D3-M4513R** [comment]
oh ffs. okay.

**Comment by C0D3-M4513R** [comment]
Just replace `Bundled_McVersion` with `KettingConstants.MINECRAFT_VERSION`?

**Comment by TonimatasDEV** [comment]
Now?

**Review by C0D3-M4513R — APPROVED** [review]

**Comment by C0D3-M4513R** [comment]
Still unsure how we should go about the update tbh.
We'll see when the other pr is ready.

**Comment by C0D3-M4513R** [comment]
I might rewrite this still. Something with the update doesn't sit right with me.
Maybe a combination of both ways.

**Review by JustRed23 — APPROVED** [review]

**Comment by C0D3-M4513R** [comment]
We are working on `dev` for making cpw work.

**Pull request closed without merging**